src/core/sub/change.cpp

```diff
@@ -313,6 +317,67 @@ namespace sub::change {
         std::optional<std::string_view> subXPath, sr::Event event, uint32_t requestId)
     {
         sr::ErrorCode error = sr::ErrorCode::Ok;
+
+        dns::DnsSearchServerList dnsSearchServers(SYSTEMD_IFINDEX);
```

Comment:
`SYSTEMD_IFINDEX` is compile time known constant, there's no need for it to be used as a runtime parameter.